<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserNotes</class>
 <widget class="QWidget" name="UserNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 40), stop:1 transparent);*/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="adiscope::StackedHomepage" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="addNote">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="saveBtn">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  	border: 0px;
  	border-radius: 4px;
  	background-color: #4a64ff;
	color: #ffffff;
  	font-size: 14px;
}

QPushButton:hover
{
background-color: #4a34ff;
}

QPushButton:disabled {
background-color: gray;
color:white;
}

QPushButton:disabled {
  background-color:  #727273;
}
</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>File path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pathWarning">
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="pathLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border-color: grey;
	color: grey;
}

QLineEdit[invalid=true] {
	border-color: red;
	color: red;
}
</string>
             </property>
             <property name="invalid" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  	border: 0px;
  	border-radius: 4px;
  	background-color: #4a64ff;
	color: #ffffff;
  	font-size: 14px;
}

QPushButton:hover
{
background-color: #4a34ff;
}

QPushButton:disabled {
background-color: gray;
color:white;
}

QPushButton:disabled {
  background-color:  #727273;
}
</string>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Note name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="7">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgba(255, 255, 255, 16);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="addWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">[selected=true] {
  	opacity: 0.6;
	border-radius: 4px;
	background-color: #141416;
}

[selected=false] {
	background-color: transparent;
}</string>
       </property>
       <property name="selected" stdset="0">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
height: 40px;
width: 40px;
border-width: 0px;
border-radius: 4px;
background-color: #4A64FF;
}

QPushButton:hover {
background-color: #4a34ff;
}

QPushButton:checked {
background-color: #4a34ff;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/menu/add.svg</normaloff>:/menu/add.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="adiscope::OscCustomScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>768</width>
          <height>130</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="notesList">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>adiscope::StackedHomepage</class>
   <extends>QStackedWidget</extends>
   <header>stacked_homepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>adiscope::OscCustomScrollArea</class>
   <extends>QScrollArea</extends>
   <header>osc_custom_scroll.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
